site/gatsby-site/src/components/cite/Tools.js

```diff
@@ -162,4 +198,123 @@ function Tools({
+        className="w-20 ml-1"
+        onChange={(event) => setDuplicateIncidentId(event.target.value)}
+        value={duplicateIncidentId}
+      />
+      {duplicateIncidentId && duplicateIncident ? (
+        <>
+          <p>
+            <LocalizedLink to={`/cite/${duplicateIncident.incident_id}`}>
+              Incident {duplicateIncident.incident_id} – {duplicateIncident.title}
+            </LocalizedLink>
+          </p>
+        </>
+      ) : duplicateIncidentLoading ? (
+        <div className="my-4">
+          <Spinner />
+        </div>
+      ) : (
+        <div className="my-8"></div>
+      )}
+      <Button
+        color="failure"
+        disabled={duplicateIncidentLoading || !duplicateIncident || deletingDuplicate}
+        onClick={async () => {
+          setDeletingDuplicate(true);
+          const reportIds = removeTypename(
+            duplicateIncidentData.incident.reports.concat(incident.reports)
+          ).map((report) => report.report_number);
+
+          let insertDuplicateResponse;
+
+          try {
+            insertDuplicateResponse = await insertDuplicate({
+              variables: {
+                duplicate: {
+                  duplicate_incident_number: incident.incident_id,
+                  true_incident_number: duplicateIncidentId,
+                },
+              },
+            });
+          } catch (e) {
+            console.error(insertDuplicateResponse);
+            alert(`Could not insert duplicate. Aborting.`);
+            return;
+          }
+
+          let updateIncidentResponse;
+
+          try {
+            updateIncidentResponse = await updateIncident({
+              variables: {
+                query: { incident_id: duplicateIncidentId },
+                set: { reports: { link: reportIds } },
+              },
+            });
+          } catch (e) {
+            console.error(updateIncidentResponse);
+            alert(`Could not transfer reports to incident ${duplicateIncidentId}.`);
+          }
+
+          let updateClassificationsResponse;
+
+          try {
+            updateClassificationsResponse = await updateClassifications({
+              variables: {
+                query: { incident_id: incident.incident_id },
+                set: { incident_id: duplicateIncidentId },
+              },
+            });
+          } catch (e) {
+            console.error(updateClassificationsResponse);
+            alert(`Could not transfer classifications to incident ${duplicateIncidentId}.`);
+          }
+
+          setDeletingDuplicate(false);
+          alert(
+            [
+              `Incident ${incident.incident_id} marked`,
+              `as duplicate of ${duplicateIncidentId}.`,
+              `Its page will updated within 24 hours.`,
+            ].join(' ')
+          );
+          window.location.pathname = '/';
+        }}
+      >
+        Remove Duplicate
+      </Button>
+    </>
+  );
+}
+
+const removeTypename = (obj) => {
```

Comment:
This triggered some alarms, but I guess it works 🤷 

And after **four** years, it looks like the Apollo team found a solution:
https://github.com/apollographql/apollo-feature-requests/issues/6#issuecomment-1507687565